specification/monitor/resource-manager/readme.md

```diff
@@ -639,16 +625,6 @@ input-file:
 - Microsoft.Insights/preview/2017-12-01-preview/metricNamespaces_API.json
 ```
 
-### Tag: package-2017-11-01-preview-only
```

Comment:
Please clean-up this tag in readme.python.md

This is the cause of failure on python CI.